docs/_managedk8s/about/index.en.md

```diff
@@ -5,5 +5,75 @@ permalink: /managedk8s/about/
 nav_order: 1000
 has_children: true
 ---
-Short description what they get 
+# About Our Managed Kubernetes
+We provide a Hosted CNCF Certified Kubernetes Cluster tailored to your needs. Unlike self-service Kubernetes offerings, we take full responsibility for creating, updating, and managing your Kubernetes clusters. Once the cluster is set up, we provide you with a kubeconfig file that grants you secure access to the cluster.
 
+# What We Manage for You
+As part of our managed Kubernetes service, we handle the following tasks:
+
+- Cluster Creation: We set up a Kubernetes cluster that meets CNCF certification standards, ensuring compatibility with the latest Kubernetes ecosystem.
+- Cluster Updates: We regularly update the cluster to the latest stable version, ensuring security and performance enhancements.
+- Cluster Management: We monitor the health and performance of the cluster control plane.
+
+By managing these core elements, we enable you to focus on your applications without worrying about the operational complexities of Kubernetes.
+
+In case anything goes wrong, you can reach out to our [24/7 support](mailto:cks.de@wiit.cloud) or, as an optional service, schedule a session with our Professional Services team to deep-dive and discuss your technical questions and architecture.
```

Comment:
We should not put that here, more like we provide a communication channel after onboarding.

dependig on the customer this could be different chanels